en/components/time_picker.md

```diff
@@ -331,6 +331,26 @@ And there we have it, a re-templated time picker with dropdown, custom actions a
 </div>
 <div class="divider--half"></div>
 
+#### Custom label
+By placing a `<label>` inside the tag of the component the default label can be changed to a custom one:
```

Comment:
marked with `igxLabel` directive

Reply:
The sentence was changed to "By placing an `igxlabel` inside a `<label>` tag nested in the component, the default label can be changed to a custom one:".